### PR TITLE
Add WZDx Feed Ingestion MVP Code

### PR DESCRIPTION
**Pull request by chuehlien**
Add code for ingesting data from Maricopa County (MCDOT) WZDx feed to ITS Work Zone Sandbox.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,138 @@
...
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for general good practices on code of conduct, and the process for submitting pull requests.
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags).
+
+## Authors
+
+See the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
```
Comment (mvs5465) [inline README.md]: Lets take out the authors section.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,138 @@
...
+
+## License
+
+This project is licensed under the Apache 2.0 License. - see the [LICENSE.md](LICENSE.md) file for details
```
Comment (mvs5465) [inline README.md]: License file does not have the .md extension, please change just to LICENSE

File: README.md [hunk 3]
```diff
@@ -1,2 +1,138 @@
...
+
+1. Fork it (https://github.com/usdot-its-jpo-data-portal/cv_pilot_ingest/fork)
+2. Create your feature branch (git checkout -b feature/fooBar)
+3. Commit your changes (git commit -am 'Add some fooBar')
+4. Push to the branch (git push origin feature/fooBar)
+5. Create a new Pull Request
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for general good practices on code of conduct, and the process for submitting pull requests.
+
+## Versioning
```
Comment (mvs5465) [inline README.md]: I would say take out the versioning section unless we want to start using releases

File: s3_helper.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        try:
+            if self.aws_profile:
+                session = boto3.session.Session(profile_name=self.aws_profile)
+            else:
+                session = boto3.session.Session()
+        except botocore.exceptions.ProfileNotFound:
+            self.print_func('Please supply a valid AWS profile name.')
+            exit()
+        except:
+            self.print_func(traceback.format_exc())
```
Comment (mvs5465) [inline s3_helper.py]: Should we exit here if another exception occurs?

File: README.md (hunk 1 above)
Comment (chuehlien) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (chuehlien) [reply README.md]: Thanks for catching!

File: README.md (hunk 3 above)
Comment (chuehlien) [reply README.md]: Removed this section as well.

File: s3_helper.py (hunk 4 above)
Comment (chuehlien) [reply s3_helper.py]: Good point. Adding additional error message and exiting here.

**Comment by chuehlien** [comment]
Incorporated the changes suggested by @mvs5465.

**Pull request merged**